### PR TITLE
W-1194107-CustomLogicUpdates-ESB

### PR DESCRIPTION
**Pull request by emileeburkeVT**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc [hunk 1]
```diff
@@ -339,13 +339,22 @@ To create an If/Else block:
 . Click the plus sign after the trigger or the last action in your flow.
 . Click *Add Action*.
 . Click *If/Else Block* to define the first branch.
-. Enter the requested criteria.
+. Enter the requested criteria. You can specify that `all conditions should be met (AND)`, `any condition can be met (OR)`, or define <<use-custom-logic,custom logic>> to customize the order and grouping of your conditions.
```
Comment (IsaacEldridge) [inline en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:342]: when the link is live does this read "define use custom logic to..." should define be cut?

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc [hunk 2]
```diff
@@ -339,13 +339,22 @@ To create an If/Else block:
...
 . Click *+* to add an action to that branch. Every branch must have an action, also called a _step_.
 . If you want to add another branch, click *Add If* and repeat the previous steps for every If/Else branch you create.
 . If you want to set criteria for how the flow executes if none of the criteria for the existing branches are met, click *Add Else* and repeat the previous steps for the final branch.
 . If you want to add additional conditions to your If/Else block, click *Add condition*.
 . After you add all the branches, click *Save*.
 
+[[use-custom-logic]]
+== Use Custom Logic
+
+Custom logic uses the row number for each condition in the If statement block of an If/Else Statement and `OR`, `AND`, and `NOT` to create complex expressions.  When you select the custom logic option, each condition is assigned a number and you can then use that number to customize the order of operations for the conditions.
```
Comment (IsaacEldridge) [inline en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:352]:
```suggestion
Custom logic uses the row number for each condition in the If statement block of an If/Else statement and `OR`, `AND`, and `NOT` to create complex expressions.  When you select the custom logic option, each condition is assigned a number and you can then use that number to customize the order of operations for the conditions.
```

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc [hunk 3]
```diff
@@ -339,13 +339,22 @@ To create an If/Else block:
...
+
+When using custom logic, all conditions defined must be referenced in the expression for the expression to be validated. For example, if there are three defined conditions and you enter “1 and (2 or 1)”, the expression is invalid because “3” is not referenced.
+
+ Using custom logic, you can have up to 20 conditions and a condition can be referenced  more than once.
```
Comment (IsaacEldridge) [inline en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:356]:
```suggestion
 Using custom logic, you can have up to 20 conditions and a condition can be referenced more than once.
```

File: en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc [hunk 4]
```diff
@@ -4,6 +4,29 @@ Use MuleSoft Composer to build connected experiences faster. You don't need to w
 
 New versions are automatically available once they are released; you don't need to perform any tasks to move to the new version.
 
+== October 27, 2022
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* You can now add custom logic, such as `and`, `or`, or `not`, in conditional statements when building an action and in If/Else Statements.
```
Comment (IsaacEldridge) [inline en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc:11]:
```suggestion
* You can now add custom logic, such as `and`, `or`, or `not`, in conditional statements when building an action and in If/Else statements.
```

File: en/salesforce/src/docs/ms_composer_release_notes_2022.adoc [hunk 5]
```diff
@@ -9,6 +9,36 @@ New versions are automatically available once they are released; you don't need
 //| Fixed an issue in NetSuite Connector in which nested fields containing the `list` type failed. | COCN-726
 //* Composer now supports the static creation of lists. You can now dynamically complete fields that are enabled for lists even if those fields do not have an existing list to map to in the flow.
 
+== October 27, 2022
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* You can now add custom logic, such as `and`, `or`, or `not`, in conditional statements when building an action and in If/Else Statements.
```
Comment (IsaacEldridge) [inline en/salesforce/src/docs/ms_composer_release_notes_2022.adoc:16]:
```suggestion
* You can now add custom logic, such as `and`, `or`, or `not`, in conditional statements when building an action and in If/Else statements.
```

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc (hunk 1 above)
Comment (emileeburkeVT) [reply en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:342]: Maybe "use custom logic" would be better

**Review by IsaacEldridge — APPROVED** [review]

**Pull request merged**